projects/core/src/lib/authorization/components/role-select/role-select.component.ts

```diff
@@ -59,28 +78,33 @@ export class RoleSelectComponent {
 	}
 
 	private roleMentee(): void {
-		this.getUser(Roles.MENTEE);
-		// toDo : Need page to add route in future
+		this.userService.saveUser(this.getUser(Roles.MENTEE)).subscribe();
+		//toDo redirect to Mentee page.
 	}
 
 	private roleExpert(): void {
-		this.getUser(Roles.EXPERT);
-		//toDo : Need page to add route in future
+		this.userService.saveUser(this.getUser(Roles.EXPERT)).subscribe();
+		//toDO redirect to Expert page
 	}
 
 	private roleRM(): void {
-		this.getUser(Roles.RM);
-		// toDo : Need page to add route in future
+		this.userService.saveUser(this.getUser(Roles.RM)).subscribe();
+		//toDo redirect to RM page
```

Comment:
plrase add .pipe(takeUntil(this.destroy$))

Reply:
Done